cli-plugins/apps/command-reference/tanzu-apps-workload-create.md

```diff
@@ -23,41 +24,45 @@ tanzu apps workload create [name] [flags]
 tanzu apps workload create my-workload --git-repo https://example.com/my-workload.git
 tanzu apps workload create my-workload --local-path . --source-image registry.example/repository:tag
 tanzu apps workload create --file workload.yaml
+tanzu apps workload create my-workload --image "-" --param-yaml maven=$"artifactId:hello-world\ntype: jar\nversion: 0.0.1\ngroupId: carto.run"
```

Comment:
I think we can/should remove the  `--image "-"` since we're removing source validation.